lib/src/components/card_rounded.dart

```diff
@@ -0,0 +1,113 @@
+import 'package:flutter/gestures.dart';
+import 'package:flutter/material.dart';
+
+import 'package:help_ukraine_widget/help_ukraine_widget.dart';
+
+/// It's a stateless widget that takes in a card and returns a card
+class CardRounded extends StatelessWidget {
+  /// A child widget for [Card].
+  final Widget? child;
+
+  /// A variable to set the background color of the card.
+  final Color? backgroundColor;
+
+  /// A variable that sets the height of the card.
+  final double? height;
+
+  /// A variable that sets the width of the card.
+  final double? width;
+
+  /// A variable that sets the close button on the card.
+  final bool withCloseButton;
+
+  /// A function that calls when close button pressed, if withCloseButton
+  /// is false, it is necessary.
+  final VoidCallback? onClose;
```

Comment:
How do we ensure that this callback is provided?

Reply:
Actually, we can withCloseButton parameter and show button only then, when  onClose is not null.